ecs.tf

```diff
@@ -15,12 +15,12 @@ locals {
       ],
       "logConfiguration" : {
         "logDriver" : "awslogs",
-        "options" : {
-          "awslogs-region" : var.aws_region,
-          "awslogs-group" : module.label.id,
-          "awslogs-stream-prefix" : module.label.id,
-          "awslogs-create-group" : "true"
-        }
+        "options" : merge({
+          "awslogs-region" : var.aws_region,                 # always required
+          "awslogs-group" : local.cloudwatch_log_group_name, # always required
+          "awslogs-stream-prefix" : module.label.id          # required for FARGATE
+          },
+        !var.cloudwatch_log_group_create ? { "awslogs-create-group" : "true" } : {}) # create ECS-managed log group
```

Comment:
`"awslogs-create-group" : "true"` has to be appended to the `logConfiguration.options` with the `merge()` function because `true` is the only accepted value for `"awslogs-create-group"`

others values (e.g. `false` or `null`) causes an error:

```
╷
│ Error: creating ECS Task Definition (prowler-shared): ClientException: Log driver awslogs option 'awslogs-create-group' should only be set to 'true', else omit option.
│ 
│   with module.prowler-shared.module.prowler-ecs-task.aws_ecs_task_definition.this,
│   on ../../../../terraform-aws-ecs-job/ecs.tf line 50, in resource "aws_ecs_task_definition" "this":
│   50: resource "aws_ecs_task_definition" "this" {
│ 
╵
```